recipe/meta.yaml

```
@@ -1,41 +1,79 @@
-{% set version = "0.4.6" %}
+{% set version = "0.5.0" %}
 
 package:
-  name: pyct
+  name: pyct-packages
   version: {{ version }}
 
+source:
+  url: https://pypi.io/packages/source/p/pyct/pyct-{{ version }}.tar.gz
+  sha256: dd9f4ac5cbd8e37c352c04036062d3c5f67efec76d404761ef16b0cbf26aa6a0
+
 build:
-  number: 0
   noarch: python
+  number: 0
 
 requirements:
-  host:
-    - python
-    - pip
-  run:
-    - python
-    - pyct-core {{ version }}
-    - pyyaml
-    - requests
-
-test:
-  imports:
-    - pyct
-    - pyct.cmd
-  requires:
-    - flake8
-    - pytest
+  build:
+    - python >=3.7
+
+outputs:
+  - name: pyct-core
+    build:
+      noarch: python
+      entry_points:
+        - pyct=pyct.__main__:main
+      script: {{ PYTHON }} -m pip install . -vv
```

Multiple outputs has a bug that forces us to define no deps and isolation. Better safe than sorry 😐

```suggestion
      script: {{ PYTHON }} -m pip install --no-deps --no-build-isolation -vv .
```